backend/routes/orders.js

```diff
@@ -9,6 +9,12 @@ router.get('/:id', orderServices.getOrderById);
 
 router.get('/userId/:id', orderServices.getOrderByUserId);
 
+router.get('/riderId/:id', orderServices.getOrderByRiderId);
+
+router.get('/delivering/:id', orderServices.updateOrderDelivering);
```

Comment:
Request method should not be GET because data is altered server side. Since this is an update function the correct method should be PATCH. Also, by REST it should be PATCH '/:id', and the state to update to is provided in the body of the request. e.g. { "status": "completed" }